src/Resources/config/services.yml

```diff
@@ -0,0 +1,50 @@
+services:
+    contao.backend.subscriber:
```

Comment:
I don't like the `contao` prefix here. It appeals being an official contao service. I suggest using `cca.event_bindings` as prefix for all services.